lib/index.js

```diff
@@ -69,7 +70,8 @@ AppsFlyer.prototype.track = function(track, fn) {
 
   // AF requires you to change this to empty string if empty
   // Also, everything in the payload should be stringified
-  isEmpty(track.properties()) ? payload.eventValue = '' : payload.eventValue = stringify(track.properties({ revenue: 'af_revenue' }));
+  var props = reject(track.properties({ revenue: 'af_revenue' }));
+  isEmpty(props) ? payload.eventValue = '' : payload.eventValue = stringify(props);
```

Comment:
This is probably more readable as
`payload.eventValue = isEmpty(props) ? '' : stringify(props);`